src/content/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model.mdx

```diff
@@ -75,11 +75,13 @@ To determine what user model you're on, from the [account dropdown](/docs/using-
 
 To determine if you can manage users on the New Relic One user model, see [Manage users](/docs/accounts/accounts-billing/new-relic-one-pricing-users/add-manage-users-groups-roles#where).
```

Comment:
I did notice that the images were broken in my local build. They still look fine on the docs site so unless you messed around with the images, you can safely ignore this comment.